src/main/java/com/exasol/adapter/dialects/exasol/ExasolLocalSqlGenerationVisitor.java

```diff
@@ -0,0 +1,42 @@
+package com.exasol.adapter.dialects.exasol;
+
+import com.exasol.adapter.AdapterException;
+import com.exasol.adapter.dialects.SqlDialect;
+import com.exasol.adapter.dialects.rewriting.SqlGenerationContext;
+import com.exasol.adapter.metadata.DataType;
+import com.exasol.adapter.sql.SqlColumn;
+/**
+ * This class generates SQL queries for the {@link ExasolSqlDialect}. It is a 'specialisation' for LOCAL mode, based on {@link ExasolSqlGenerationVisitor}.
```

Comment:
Please add a note how it is specialized.

Reply:
Did that.